www/_includes/footer.html

```diff
@@ -1,6 +1,6 @@
 <div class="blue-divider"></div>
 <footer>
-    <div class="container" id="footer">
```

Comment:
Great catch! It makes the footer look a bit strange on other pages. I'll come up with a fix.